notebooks/data-sources/github/thoth_PR_data_extraction.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>Can we remove the installs from here and add packages to the pipfile and lock? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>